modules/core/src/commonMain/kotlin/fr/acinq/lightning/db/PaymentsDb.kt

```diff
@@ -465,103 +480,82 @@ data class LightningOutgoingPayment(
 
 sealed class OnChainOutgoingPayment : OutgoingPayment() {
     abstract override val id: UUID
-    abstract val miningFees: Satoshi
+    abstract val miningFee: Satoshi
     abstract val channelId: ByteVector32
     abstract val txId: TxId
+    abstract val liquidityPurchase: LiquidityAds.Purchase?
```

Comment:
Including the `liquidityPurchase` here is a bit of an abuse because `ChannelCloseOutgoingPayment` logically cannot have one, but it is acceptable.